src/utils.js

```diff
@@ -142,3 +144,14 @@ const humanFileSize = (size) => {
   const unit = ['byte', 'kilobyte', 'megabyte', 'gigabyte', 'terabyte'][index];
   return {size: newSize, unit};
 };
+
+
+export const initialiseSentry = (sentryDns) => {
+  if (sentryDns) {
+    Sentry.init({
+      dsn: sentryDns,
```

Comment:
I forgot to spec in the tasks, but it's nice to also set the environment param so we know which client/environment the issue is about.